stdlib/html/parser.pyi

```diff
@@ -28,3 +29,17 @@ class HTMLParser(ParserBase):
     def parse_pi(self, i: int) -> int: ...  # undocumented
     def parse_starttag(self, i: int) -> int: ...  # undocumented
     def set_cdata_mode(self, elem: str) -> None: ...  # undocumented
+    def parse_comment(self, i: int, report: int = ...) -> int: ...  # undocumented
+    def parse_declaration(self, i: int) -> int: ...  # undocumented
+    def parse_marked_section(self, i: int, report: int = ...) -> int: ...  # undocumented
+    def updatepos(self, i: int, j: int) -> int: ...  # undocumented
```

Comment:
```suggestion
```

These redefinitions are all unnecessary, since the signatures of these methods is the same as on the parent class.

Reply:
figured this was the case, but wanted to play it safe. Thanks for clarifying, removing now!